patches/server/1000-Increase-vanilla-mushroom-spread-limit.patch

```diff
@@ -0,0 +1,29 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: grayr0ot <thisis41@protonmail.com>
+Date: Wed, 9 Aug 2023 22:04:15 +0200
+Subject: [PATCH] Increase vanilla mushroom spread limit
+
+
+diff --git a/src/main/java/net/minecraft/world/level/block/MushroomBlock.java b/src/main/java/net/minecraft/world/level/block/MushroomBlock.java
+index 5238b23cd3bca21e2b14c9be15699b95ad267e24..1afa460b388e1a772b278ac6710adc211fd5f2be 100644
+--- a/src/main/java/net/minecraft/world/level/block/MushroomBlock.java
++++ b/src/main/java/net/minecraft/world/level/block/MushroomBlock.java
+@@ -40,10 +40,17 @@ public class MushroomBlock extends BushBlock implements BonemealableBlock {
+     @Override
+     public void randomTick(BlockState state, ServerLevel world, BlockPos pos, RandomSource random) {
+         if (random.nextFloat() < (world.spigotConfig.mushroomModifier / (100.0f * 25))) { // Spigot - SPIGOT-7159: Better modifier resolution
+-            int i = 5;
++            // Paper start - Increase vanilla mushroom spread
++            int mushroomSpreadLimit = world.paperConfig().environment.mushroom.spreadLimit.or(5);
```

Comment:
Leaving the original i = 5 and using the i as the default in the .or might look a bit cleaner, but generally looks pretty good.